ci-operator/step-registry/nutanix/validate-credential-rotation/nutanix-validate-credential-rotation-commands.sh

```
@@ -0,0 +1,380 @@
+# Disable tracing for credential operations
+set +x
+source /tmp/nutanix-creds/secrets.sh
+
+# Set up new credentials (either from env vars or simulate with existing)
+if [[ -z "${NEW_NUTANIX_USERNAME:-}" ]]; then
+  echo "No NEW_NUTANIX_USERNAME provided, simulating rotation with existing credentials"
+  NEW_NUTANIX_USERNAME="${prism_central_username}"
+  NEW_NUTANIX_PASSWORD="${prism_central_password}"
+  SIMULATED_ROTATION=true
+else
+  NEW_NUTANIX_USERNAME="${NEW_NUTANIX_USERNAME}"
```

Shellcheck (0.11.0)

[info] This variable is assigned to itself, so the assignment does nothing.

(SC2269)